src/masternodes/icxorder.h

```diff
@@ -239,13 +259,13 @@ struct CICXSubmitDFCHTLCMessage : public CICXSubmitDFCHTLC {
 class CICXSubmitEXTHTLC
 {
 public:
-    static const int STATUS_OPEN;
-    static const int STATUS_EXPIRED;
+    static const uint8_t STATUS_OPEN;
 
     // This tx is acceptance of the offer and evidence of HTLC on external chain in the same time. It is a CustomTx on DFC chain
     //! basic properties
     uint256 offerTx; // txid for which offer is this HTLC
     CAmount amount;
+    CScript receiveAddress;
     uint256 hash; 
     std::string htlcscriptAddress;
```

Comment:
Unused ?

Reply:
Used in case of external type of order - BTC order comes first.

But, I realized recently that only one address is used for all tokens and I wanted to refactor the flow so that in order will be token address and optional btc pubkey for BTC/DFI order and in offer to have also token address and optional BTC pubkey for DFI/BTC order. Then there is no need for addresses in HTLCs!